content/applications/finance/fiscal_localizations/chile.rst

```diff
@@ -5,109 +5,124 @@ Chile
 
 - `VIDEO WEBINAR OF CHILEAN LOCALIZATION: INTRO AND DEMO <https://youtu.be/BHnByZiyYcM>`_.
 - `VIDEO WEBINAR OF DELIVERY GUIDE <https://youtu.be/X7i4PftnEdU>`_.
+- `TUTORIALS <https://youtube.com/playlist?list=PL1-aSABtP6AB6UY7VUFnVgeYOaz33fb4P>`_.
 
 Introduction
 ============
 
-The Chilean localization has been improved and extended in Odoo v13. In this version, the next
+The Chilean localization has been improved and extended in Odoo 16. In this version, the next
 modules are available:
 
 - **l10n_cl:** Adds accounting features for the Chilean localization, which represent the minimal
   configuration required for a company to operate in Chile and under the SII (Servicio de Impuestos
   Internos) regulations and guidelines.
-
 - **l10n_cl_edi:** Includes all technical and functional requirements to generate and receive
-  Electronic Invoice via web service, based on the SII regulations.
-
+  electronic invoice via web service, based on the :abbr:`SII (Servicio de Impuestos Internos)`
+  regulations.
 - **l10n_cl_reports:** Adds the reports Propuesta F29 y Balance Tributario (8 columnas).
+- **l10n_cl_boletas:** Includes all technical and functional requirements to generate and receive
+  electronic receipts via web service, based on the :abbr:`SII (Servicio de Impuestos Internos)`
+  regulations.
+- **l10n_cl_edi_stock:** Includes all technical and functional requirements to generate delivery
+  guides via web service, based on the :abbr:`SII (Servicio de Impuestos Internos)` regulations.
+- **l10n_cl_edi_exports:** Includes all technical and functional requirements to generate electronic
+  invoices for exporting goods, based on the :abbr:`SII (Servicio de Impuestos Internos)` and
+  customs regulations.
 
 Configuration
 =============
 
 Install the Chilean localization modules
 ----------------------------------------
 
-For this, go to *Apps* and search for Chile. Then click *Install* in the module **Chile
-E-invoicing**. This module has a dependency with **Chile - Accounting**. In case this last
-one is not installed, Odoo installs it automatically with E-invoicing.
+For this, go to :menuselection:`Apps` and search for `Chile`. Then click :guilabel:`Install` for all
+modules.
 
-.. image:: chile/Chile01.png
+.. image:: chile/clile-l10n-modules.png
    :align: center
-   :alt: Chilean module to install on Odoo.
+   :alt: Install Chilean localization modules.
 
 .. note::
-   When you install a database from scratch selecting Chile as country, Odoo will automatically
-   install the base module: Chile - Accounting.
+   When a database is created from scratch and selecting :guilabel:`Chile` as the country, Odoo will
+   automatically install the base module: *Chile - Accounting*. The module *l10n_cl_edi_stock* is
+   dependent on the *Inventory* application.
 
+Company settings
+~~~~~~~~~~~~~~~~
 
-Company Settings
-----------------
+Once the modules are installed, the first step is to set up the company data.
```

Comment:
```suggestion
Once the modules are installed, the first step is to set up the company data. This can be done in General Settings of Odoo, in the Companies section.